### ISSUE TITLE
gotoAnything command removed?

### ISSUE THREAD

**Issue by thejohnfreeman** [laurent22/joplin#5139]
## Environment

Joplin version: 2.0.11
Platform: Linux
OS specifics: Ubuntu 21.04

## Steps to reproduce

I don't know how to test this without having a plugin to edit. In a plugin, I can write `joplin.commands.execute('gotoAnything')` and get this error:

```
Uncaught (in promise) Error: Command not found: gotoAnything. Make sure the declaration has been registered.
    at CommandService.commandByName (/tmp/.mount_JoplinKP…ommandService.js:85)
    at CommandService.<anonymous> (/tmp/.mount_JoplinKP…mmandService.js:137)
    at Generator.next (<anonymous>)
    at /tmp/.mount_JoplinKP…CommandService.js:8
    at new Promise (<anonymous>)
    at __awaiter (/tmp/.mount_JoplinKP…CommandService.js:4)
    at CommandService.execute (/tmp/.mount_JoplinKP…mmandService.js:136)
    at JoplinCommands.<anonymous> (/tmp/.mount_JoplinKP…oplinCommands.js:84)
    at Generator.next (<anonymous>)
    at /tmp/.mount_JoplinKP…JoplinCommands.js:8
```

But in the normal dev console, `commands` is not a property of `joplin`. Instead, I can try `joplin.commandService.commands_.gotoAnything.runtime.execute()`, except that `gotoAnything` is not a property of `joplin.commandService.commands_`.

The command still appears in the [directory](https://github.com/laurent22/joplin/tree/5ee2ffe174369d23db4eaaa23e353e8b3b08bb71/packages/app-desktop/gui/MainScreen/commands) linked from the [documentation](https://joplinapp.org/api/references/plugin_api/classes/joplincommands.html) as "Main Screen Commands"

## Describe what you expected to happen

This was working in the last version I had installed, which was 1.7.3. I expect it to open the "Goto Anything" modal search box.

**Comment by roman-r-m** [comment]
Possibly related: https://github.com/laurent22/joplin/commit/00b39e46715c2afbb971fa7a40c5e43705a7c636

**Comment by laurent22** [comment]
This command is in version 2.1.7

**Comment by thejohnfreeman** [comment]
I just tested with 2.1.7, and that does not appear to be the case. The fix that @roman-r-m linked was committed 4 days ago, and 2.1.7 was released 5 days ago. Maybe it will be in the next release?

**Comment by laurent22** [comment]
Yes it should be in the next pre-release then.

**Comment by thejohnfreeman** [comment]
The fix was included in release [2.2.1](https://github.com/laurent22/joplin/releases/tag/v2.2.1).
